content/questions/admin/question-051.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "How does the role of an outside collaborator differ from that of a member within a GitHub organization?"
+archetype: "questions"
+title: "Question 051"
+draft: false
+---
+
+> https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization
+1. [x] Outside collaborators are granted access to specific repositories rather than the entire organization.
```

Comment:
Same comment as i put in q49

Members dont have access to all repositories either.